test/test_other.py

```diff
@@ -8666,7 +8666,7 @@ def test(check, extra):
     self.assertNotContained(WARNING, proc.stderr)
 
   def test_full_js_library(self):
-    self.run_process([EMCC, test_file('hello_world.c'), '-sSTRICT_JS', '-sINCLUDE_FULL_LIBRARY'])
+    self.run_process([EMCC, test_file('hello_world.c'), '-sSTRICT_JS', '-sINCLUDE_FULL_LIBRARY', '-sASYNCIFY'])
```

Comment:
Maybe add a comment here? Otherwise asyncify seems odd.

Reply:
I refactored this to use `@paramaterized` and now run them all under older node.